site/app/views/grading/ElectronicGraderView.php

```diff
@@ -50,7 +50,7 @@ public function statusPage(
     ) {
 
         $peer = false;
-        if ($gradeable->isPeerGrading() && $this->core->getUser()->getGroup() == User::GROUP_STUDENT) {
+        if ($gradeable->isPeerGrading()) {//&& $this->core->getUser()->getGroup() == User::GROUP_STUDENT) {
```

Comment:
Delete the commented out part here, or add a comment as to why you're leaving it commented out.